cypress/integration/checkout/checkout.multipiece.items.spec.js

```diff
@@ -0,0 +1,152 @@
+                    status: { name: 'Available' },
+                    descriptionOfPieces: descriptionOfPiece,
+                    permanentLoanType: { id: Cypress.env('loanTypes')[0].id },
+                    materialType: { id: Cypress.env('materialTypes')[0].id },
+                  },
+                  {
+                    barcode: SECOND_ITEM_BARCODE,
+                    numberOfPieces: quantityPiecesForSecondItem,
+                    status: { name: 'Available' },
+                    descriptionOfPieces: descriptionOfPiece,
+                    permanentLoanType: { id: Cypress.env('loanTypes')[0].id },
+                    materialType: { id: Cypress.env('materialTypes')[0].id },
+                  },
+                  {
+                    barcode: THIRD_ITEM_BARCODE,
+                    numberOfPieces: quantityPiecesForThirdItem,
+                    status: { name: 'Available' },
+                    descriptionOfPieces: descriptionOfPiece,
+                    numberOfMissingPieces: quantityOfMissingPieces,
+                    missingPieces: missingPieceDescription,
+                    permanentLoanType: { id: Cypress.env('loanTypes')[0].id },
+                    materialType: { id: Cypress.env('materialTypes')[0].id },
+                  },
+                  {
+                    barcode: FOURTH_ITEM_BARCODE,
+                    numberOfPieces: quantityPiecesForFourthItem,
+                    status: { name: 'Available' },
+                    numberOfMissingPieces: quantityOfMissingPieces,
+                    missingPieces: missingPieceDescription,
+                    permanentLoanType: { id: Cypress.env('loanTypes')[0].id },
+                    materialType: { id: Cypress.env('materialTypes')[0].id },
+                  },
+                ],
+              ],
+            });
+          });
+      });
+  });
+
+  after('Delete all data', () => {
+    cy.getInstance({ limit: 1, expandAll: true, query: `"items.barcode"=="${FIRST_ITEM_BARCODE}"` })
+      .then(instance => {
+        instance.items.forEach(item => {
+          cy.deleteItem(item.id);
+        });
+        cy.deleteHoldingRecord(instance.holdings[0].id);
+        cy.deleteInstanceApi(instance.id);
+      });
+    SwitchServicePoint.changeServicePointPreference();
+    cy.deleteServicePoint(servicePoint.body.id);
+    cy.deleteUser(user.userId);
+  });
+
+  it('C591 Check out: multipiece items', { tags: [TestTypes.smoke] }, () => {
+    const dash = '-';
+
+    cy.visit(TopMenu.checkOutPath);
+    CheckOutActions.checkIsInterfacesOpened();
+    CheckOutActions.checkOutItem(userBarcode, FIRST_ITEM_BARCODE);
+    CheckOutActions.checkPatronInformation(user.username, userBarcode);
+    CheckOutActions.checkConfirmMultipieceCheckOutModal();
+    CheckOutActions.checkOutItem(userBarcode, SECOND_ITEM_BARCODE);
+    ConfirmMultipieceCheckOutModal.checkIsModalConsistOf(instanceTitle, quantityPiecesForSecondItem, descriptionOfPiece);
```

Comment:
think about better naming. it's difficult to cross your code with steps in the TC in testrail